Source/Core/Core/ConfigManager.cpp

```diff
@@ -592,6 +593,7 @@ void SConfig::LoadNetworkSettings(IniFile& ini)
   network->Get("SSLVerifyCertificates", &m_SSLVerifyCert, true);
   network->Get("SSLDumpRootCA", &m_SSLDumpRootCA, false);
   network->Get("SSLDumpPeerCert", &m_SSLDumpPeerCert, false);
+  network->Get("NetworkTimeout", &m_NetworkTimeout, 190);
```

Comment:
It might be worth documenting that this value comes from a hwtest? Otherwise 190 just looks like a random timeout value.